### DISCUSSION TITLE
Error when initializing

### DISCUSSION THREAD

**Discussion by krb-nantes** [idaholab/moose#25098] [Q&A General] [answered]
Hi everyone, I am having an issue when running an RIA case. I first ran the standard steady state case and use the final check point as restart for the RIA file. On Tuesday I ran the RIA file and it worked fine, but it hit the walltime and did not finished, then when I tried to ran again with a longer walltime I started having the following error message when initializing:

```
Finished Setting Up                                                                      [  2.63 s] [  300 MB]
libMesh terminating:
map_find() error: key "residual_temperature" not found in file ../src/systems/system.C on line 1448
[0] ./include/libmesh/utility.h, line 155, compiled Jun 22 2023 at 09:32:30
```

I already checked both files and there is no difference on their modules. I also updated and recompiled my petsc and libmesh.
Just for information, I am using the BISON code, and already checked with their team, but no solution so far.

Anyone has any suggestions of what may cause this issue?

**Comment by lindsayad** [comment]
What does RIA mean?

**Reply by krb-nantes** [reply]
Hi @lindsayad , sorry about that, RIA stands for Reactivity-Initiated Accident. My simulation is of a full length rod, first I do an irradiation and then apply a power pulse that represents the RIA.

**Reply by lindsayad** [reply]
Are you restarting your simulation?

**Reply by krb-nantes** [reply]
Yes, I am using on the second simulation a restart from the final time step of the first simulation.

**Reply by lindsayad** [reply]
are your variables the same between the previous and new simulation? They will need to be

**Reply by krb-nantes** [reply]
Yes, variables and models are all the same, the only changes are on [Problem] block, that I added “restart_file_base” and at the [Executioner], I changed the end_time

**Comment by GiudGiud** [comment, marked as answer]
Did you have a debug option to show the residual in one simulation and not the other? 

It looks like this was added in the second one.

**Reply by krb-nantes** [reply]
@GiudGiud , I changed the Debug options and it worked. It still doesn't make sense to me, as it was working before without the debug options, but I guess it will be sufficient. Thanks!

**Reply by GiudGiud** [reply]
The debug option creates an auxiliary variable. This variable is not present in the initial simulation
